nautobot/docs/plugins/development.md

```diff
@@ -1146,3 +1146,147 @@ try:
 except ImportError:
     pass
 ```
+
+## NautobotUIViewSet
```

Comment:
Should this be moved up so that it falls under the `Adding Web UI Views` section heading?